torch_xla/csrc/translator.cpp

```diff
@@ -489,6 +503,17 @@ XlaComputationInOut XlaTranslator::BuildComputationProgram(
         cctx.AddNodeOp(node, xla_output);
         break;
       }
+      case aten::size: {
+        CHECK_EQ(node->inputs().size(), 1);
+        const auto shape_sizes = XlaHelpers::ShapeSizes(
+            XlaHelpers::ShapeOfXlaOp(cctx.OpForInput(node, 0)));
+        const auto it_ok = size_op_values_tracking.insert(
+            std::pair<size_t, XlaComputationInOut::ShapeSizes>{
+                node->output(0)->unique(), shape_sizes});
+        CHECK(it_ok.second);
```

Comment:
XLA_CHECK* ... with some info.

Reply:
I wanted to be consistent here with the rest of the translator, should we do this in a follow-up for the entire translator instead?